src/use_case/new_game/NewGameOutputData.java

```diff
@@ -1,15 +1,23 @@
 package use_case.new_game;
 import entity.Game;
 
+import java.util.HashMap;
+
 public class NewGameOutputData {
 
-    private final Game newGame;
+    private final HashMap werewolves;
+    private final HashMap villagers;
 
     public NewGameOutputData(Game newGame){
-        this.newGame = newGame;
+        this.werewolves = newGame.getAliveWerewolves();
```

Comment:
This probably should be fine? I'm assuming that you're going to use the hashmaps to show a player's role in the presenter.